unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs

```diff
@@ -144,7 +144,8 @@ private void Update_Trigger(InputAction_Trigger[] triggerTimeActions)
 #endif
                     break;
                 case DCLAction_Trigger.ToggleNavMap:
-                    InputProcessor.FromKey(action, KeyCode.M, modifiers: InputProcessor.Modifier.FocusNotInInput);
+                    if (!DataStore.i.common.isWorld.Get())
```

Comment:
It looks like a hack but we can live with it

Reply:
Agree, I didn't like it as well. I saw several similar code-lines in that InputController and decided to go with it. Also because this is a temporary solution as I understand, until we clarify what to do with the Map in the World